docker/src/main/java/brooklyn/entity/container/docker/DockerHostSshDriver.java

```diff
@@ -34,7 +27,6 @@
 import com.google.common.base.Stopwatch;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Lists;
-
```

Comment:
Need to fix imports. I can provide configuration for Eclipse, not sure about IntelliJ